patches/chrome-browser-resources-settings-settings_menu-settings_menu.ts.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/browser/resources/settings/settings_menu/settings_menu.ts b/chrome/browser/resources/settings/settings_menu/settings_menu.ts
+index d756de3d8e76cb10c4ad04184411fce87f7a7c86..83e4a8884661e88452beccae560dbee513886eb2 100644
+--- a/chrome/browser/resources/settings/settings_menu/settings_menu.ts
++++ b/chrome/browser/resources/settings/settings_menu/settings_menu.ts
+@@ -147,7 +147,7 @@ export class SettingsMenuElement extends SettingsMenuElementBase {
+    * `element.getAttribute('href')`. Using `element.href` will not work as it
+    * would pass the entire URL instead of just the path.
+    */
+-  private setSelectedPath_(path: string) {
++  public setSelectedPath_(path: string) {
```

Comment:
We call this function from our settings menu scroll handler in order to properly highlight items in the settings menu, but it's private so it won't compile with type checking enabled. Note: This function was also renamed to `setSelectedPath_` upstream half a year ago, but since our type checking was disabled we never caught that change.

Reply:
I think you should be able to just remove `private` in the patch - methods are public by default in JS

Additionally, you can probably avoid a patch if you use an `as any` cast - the `private` is a compile time only feature, so at runtime the method will exist and be accessible.

Up to you which you think is better :smile:

Reply:
Nice, I'll use that approach to remove this patch then! I think it's best to save the patching for where it's absolutely required.